draft-ietf-tls-iana-registry-updates.md

```diff
@@ -323,6 +323,18 @@ Note:
 Track RFCs.  Supported Groups marked as "No" are not; supported groups
 marked "No" range from "good" to "bad" from a cryptographic standpoint.
 
+Despite the following behavior being crazy, experience has shown that
+some customers use the IANA registry as checklist against which to
+measure an implemention's completeness and some implementers blindly
+implement cipher supported.  Therefore, IANA [SHALL add/has added] the
+following warning to the registry:
+ 
+WARNING:
+: Supported groups will be broken or weakened over time. Blindly
```

Comment:
Would it be better to say:

"Cryptographic algorithms and parameters, such as supported groups, will be broken or weakened over time.

Reply:
Yes.